packages/@webex/plugin-authorization-browser-first-party/src/authorization.js

```diff
@@ -230,6 +232,25 @@ const Authorization = WebexPlugin.extend({
       });
   },
 
+  /**
+   * Extracts the orgId from the returned code from idbroker
+   * @instance
+   * @memberof AuthorizationBrowserFirstParty
+   * @param {String} code
+   * @private
+   * @returns {String}
+   */
+  _extractOrgIdFromCode(code) {
+    let orgId;
+
+    try {
+      orgId = code.split('_')[2];
```

Comment:
Can we link to the idbroker spec which guarantees this return value in this way?

Reply:
I've added a link to the spec for this